gatsby/package.json

```diff
@@ -11,6 +11,7 @@
     "@mdx-js/mdx": "^1.4.4",
     "@mdx-js/react": "^1.4.4",
     "bootstrap": "^3.4.1",
+    "broken-link-checker": "github:alexlouden/broken-link-checker",
```

Comment:
TODO: Update to use the main fork once [this PR](https://github.com/stevenvachon/broken-link-checker/pull/137) is merged.